[0m[[0m[31merror[0m] [0m[0morg.postgresql.util.PSQLException: ERROR: relation "SUPPLIERS" already exists[0m
[0m[[0m[31merror[0m] [0m[0m	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:488)[0m
[0m[[0m[31merror[0m] [0m[0m	at com.zaxxer.hikari.pool.ProxyPreparedStatement.execute(ProxyPreparedStatement.java:44)[0m
[0m[[0m[31merror[0m] [0m[0m	at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.execute(HikariProxyPreparedStatement.java)[0m
[0m[[0m[31merror[0m] [0m[0m	at slick.jdbc.JdbcActionComponent$SchemaActionExtensionMethodsImpl$$anon$5.$anonfun$run$5(JdbcActionComponent.scala:287)[0m
[0m[[0m[31merror[0m] [0m[0m	at slick.jdbc.JdbcActionComponent$SchemaActionExtensionMethodsImpl$$anon$5.$anonfun$run$5$adapted(JdbcActionComponent.scala:287)[0m
[0m[[0m[31merror[0m] [0m[0m	at slick.jdbc.JdbcBackend$SessionDef.withPreparedStatement(JdbcBackend.scala:425)[0m
[0m[[0m[31merror[0m] [0m[0m	at slick.jdbc.JdbcBackend$SessionDef.withPreparedStatement$(JdbcBackend.scala:420)[0m
[0m[[0m[31merror[0m] [0m[0m	at slick.jdbc.JdbcBackend$BaseSession.withPreparedStatement(JdbcBackend.scala:489)[0m
[0m[[0m[31merror[0m] [0m[0m	at slick.jdbc.JdbcActionComponent$SchemaActionExtensionMethodsImpl$$anon$5.$anonfun$run$4(JdbcActionComponent.scala:287)[0m
[0m[[0m[31merror[0m] [0m[0m	at slick.jdbc.JdbcActionComponent$SchemaActionExtensionMethodsImpl$$anon$5.$anonfun$run$4$adapted(JdbcActionComponent.scala:287)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.collection.immutable.Vector.foreach(Vector.scala:1856)[0m
[0m[[0m[31merror[0m] [0m[0m	at slick.jdbc.JdbcActionComponent$SchemaActionExtensionMethodsImpl$$anon$5.run(JdbcActionComponent.scala:287)[0m
[0m[[0m[31merror[0m] [0m[0m	at slick.jdbc.JdbcActionComponent$SchemaActionExtensionMethodsImpl$$anon$5.run(JdbcActionComponent.scala:285)[0m
[0m[[0m[31merror[0m] [0m[0m	at slick.jdbc.JdbcActionComponent$SimpleJdbcProfileAction.run(JdbcActionComponent.scala:28)[0m
[0m[[0m[31merror[0m] [0m[0m	at slick.jdbc.JdbcActionComponent$SimpleJdbcProfileAction.run(JdbcActionComponent.scala:25)[0m
[0m[[0m[31merror[0m] [0m[0m	at slick.dbio.DBIOAction$$anon$4.$anonfun$run$3(DBIOAction.scala:239)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.collection.immutable.Vector.foreach(Vector.scala:1856)[0m
[0m[[0m[31merror[0m] [0m[0m	at slick.dbio.DBIOAction$$anon$4.run(DBIOAction.scala:239)[0m
[0m[[0m[31merror[0m] [0m[0m	at slick.dbio.DBIOAction$$anon$4.run(DBIOAction.scala:237)[0m
[0m[[0m[31merror[0m] [0m[0m	at slick.basic.BasicBackend$DatabaseDef$$anon$3.liftedTree1$1(BasicBackend.scala:276)[0m
[0m[[0m[31merror[0m] [0m[0m	at slick.basic.BasicBackend$DatabaseDef$$anon$3.run(BasicBackend.scala:276)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.lang.Thread.run(Thread.java:834)[0m
[0m[[0m[31merror[0m] [0m[0m(Compile / [31mrun[0m) org.postgresql.util.PSQLException: ERROR: relation "SUPPLIERS" already exists[0m
